### DISCUSSION TITLE
Recurring and scheduled sends

### DISCUSSION THREAD

**Discussion by ATXMJ** [trustwallet/wallet-core#1280] [Ideas & Feature request]
Would it be feasible to create notifications for recurring and scheduled sends?

"Approving" a recurring send could probably be as simple as

- Click the notification for the recurring send
- Authenticate
- Click approve (or cancel)

In my opinion, this would help draw in a lot of potential users, as not many P2P payment platforms (including P2P fiat platforms) even offer this feature.

For a little more detail:
- User can create both recurring and single scheduled sends
- User can set end date for recurring send 
- User can select amount for recurring/scheduled send in either USD or coin being sent 
- User should be re-prompted periodically (maybe at a configurable rate) until they have explicitly either approved or cancelled the send, even if they have dismissed/cleared the notification
- User can disable/suspend a recurring send without deleting it

**Comment by hewigovens** [comment]
it sounds interesting, but we will unlikely add this feature

**Reply by ATXMJ** [reply]
Care to explain why?
